src/PanelTraits/AutoSet.php

```diff
@@ -27,6 +41,19 @@ public function setFromDb()
                 'values'     => [],
                 'attributes' => [],
             ];
+
+            if(!is_null($crudTypes) && isset($crudTypes[$field])) {
+                if(is_array($crudTypes[$field])) {
+                    foreach($crudTypes[$field] as $key => $value) {
+                        $new_field[$key] = $value;
+                    }
+                } else {
+                    $new_field['type'] = $crudTypes[$field];
+                }
+            }
+
+            if($new_field['type'] == 'check') { $new_field['type'] = 'checkbox'; }
```

Comment:
This shouldn't be on one line and we're missing a space between the if anyway :)